### ISSUE TITLE
Change “bump” to “update”

### ISSUE THREAD

**Issue by vandrijevik** [dependabot/feedback#78]
I believe that the word update is more familiar and reads better than bump, and I suggest changing the use of bump to update. Dependabot already uses the word update in several places:

- The dropdown menu for frequency contains *Live updates* (and not live bumps)
- The modal for a repository’s configuration is titled *Edit Update Config*, and all of the language in this modal uses update
- The modal for account settings uses update in all instances except for one (which is currently a bit confusing since it is the lone exception)

In addition to bringing consistency to Dependabot itself, using update will bring dependabot in line with the dependency managers it supports (for example, in Ruby both `gem` and `bundle` have the command `update`, and neither has `bump`).

Finally, pull requests titles like *Update nokogiri from 1.8.3 to 1.8.4* are much more understandable than *Bump nokogiri from 1.8.3 to 1.8.4* for everyone, including non-technical audiences.

Thanks for your consideration!

**Comment by greysteil** [comment]
Thanks for this @vandrijevik - really interesting feedback.

There's some history behind Dependabot's use of the word `bump`: in the very early days of the project the whole app was called Bump. It's also the term I used to use when I was creating dependency update PRs myself ages ago. I'm slightly sentimental about it as a result, and quite like that it gives Dependabot a bit of character.

That said, I try to keep the wording of Dependabot text as simple and accessible as possible, and don't want using `bump` to get in the way of that.

I've updated a the following screens in the dashboard:
- the dropdown menu of frequency options now says `<frequency> updates` for each frequency
- the account settings modal now says `update` instead of `bump`
- the dashboard index view now says `Last checked` instead of `Last bumped`, and `Updating daily` instead of `Bumping daily`

I think all of the above are clearer - thanks for the feedback and pushing me to improve them.

I've kept a few references to `bump`, because I think they're clear in context and not too awkwardly phrased:
- the `Bump now` button still uses bump. `Check now` was an option, but didn't feel sufficiently better
- I've kept the PR titles using `Bump`, because I quite like that they're distinctive. `Update` is a bit more generic, so less immediately identifiable as a dependency change

Does the above make sense to you? I want to walk the line of making Dependabot easily accessible, but also keeping its character.
